core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts

```diff
@@ -99,6 +99,7 @@ export class DotEditContentHtmlService {
     private remoteRendered: boolean;
     private askToCopy = true;
 
+    private readonly origin = window.location.origin;
```

Comment:
I think you can use `location.origin`

Reply:
Better let the `window`

Reply:
Why not use `@Inject(DOCUMENT) private document: Document` to get the location, not is a good practice access direct to `window` in Angular.
```
constructor(@Inject(DOCUMENT) private document: Document) {
    // Obtener la URL actual utilizando el objeto document
    const href = this.document.location.href;
    // Crear un objeto URL basado en la URL actual
    const url = new URL(href);
    // Obtener el origen de la URL
    const origin = url.origin;

    console.log(origin); // Esto imprimirá el origen de la URL como 'http://example.com'
  }
```
too much?

Reply:
Already done!